packages/auth/demo/public/sample-config.js

```diff
@@ -1,4 +1,5 @@
 /*
+* @license
```

Comment:
Nit: The line is misaligned.

Reply:
Fixed script to match indent of line below when inserting license tags.